src/navigators/createDrawerNavigator.js

```diff
@@ -41,6 +41,10 @@ const DefaultDrawerConfig = {
   drawerPosition: 'left',
   drawerBackgroundColor: 'white',
   useNativeAnimations: true,
+  drawerType: 'front',
```

Comment:
I know it's from the gesture handler repo, but it's unclear what `drawerType` means

Reply:
`drawerType` is one of many existing passthrough props, all of which have the same names as the props they get passed through to on `DrawerLayout`. I'm inclined to maintain that, so that there's no ambiguity about what happens with those props.